pkg/apis/edgesite/v1alpha1/types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	cloudcoreconfig "github.com/kubeedge/kubeedge/pkg/apis/cloudcore/v1alpha1"
+	edgecoreconfig "github.com/kubeedge/kubeedge/pkg/apis/edgecore/v1alpha1"
+)
+
+const (
+	// DataBaseDriverName is sqlite3
+	DataBaseDriverName = "sqlite3"
+	// DataBaseAliasName is default
+	DataBaseAliasName = "default"
+	// DataBaseDataSource is edge.db
+	DataBaseDataSource = "/var/lib/kubeedge/edgesite.db"
+)
+
+// EdgeSideConfig indicates the edgesite config which read from edgesite config file
+type EdgeSideConfig struct {
+	metav1.TypeMeta
+	// DataBase indicates database info
+	// +Required
+	DataBase *edgecoreconfig.DataBase `json:"database,omitempty"`
+	// EdgeSite indicates edgesite info
+	// +Required
+	EdgeSite *EdgeSite `json:"edgeSite,omitempty"`
```

Comment:
use NodeName directly here?

Reply:
Considering future expansion, the `NodeName`  field is put into the `EdgeSite `structure, and now the `EdgeSite` structure name is changed to `EdgeSiteDetail`

Reply:
done